### PR TITLE
Changes to run consul multinode

### PR DESCRIPTION
**Pull request by ozlevka-ericom**
By Issue https://github.com/EricomSoftwareLtd/SB/issues/924

### CODE REVIEW

File: Setup/docker-compose_dev.yml [hunk 1]
```diff
@@ -75,16 +75,15 @@ services:
             cpus: "1"
       volumes:
          - elastic:/var/lib
-    
-  consul:
-    image: securebrowsing/shield-configuration:latest
+
+  consul-server:
+    image: securebrowsing/shield-configuration:multi
```
Comment (BenyH) [inline Setup/docker-compose_dev.yml]: Why it is multi and not latest?

File: Setup/docker-compose_dev.yml [hunk 2]
```diff
@@ -98,12 +97,37 @@ services:
...
           cpus: "1"
     environment:
-        - "CONSUL_BIND_INTERFACE=eth2"
+        - "CONSUL_BIND_INTERFACE=eth0"
         - "NUMBER_OF_EXPECTED=1"
     logging:
       driver: syslog
       options:
         syslog-address: udp://${SYS_LOG_HOST}:5014
+  consul:
```
Comment (BenyH) [inline Setup/docker-compose_dev.yml]: Should we rename it to consul agent?

File: Setup/docker-compose_dev.yml [hunk 3]
```diff
@@ -75,16 +75,15 @@ services:
...
+    image: securebrowsing/shield-configuration:multi
     networks:
          - shield
-    ports:
-        - "8500:8500"
     deploy:
       mode: replicated
-      replicas: 1
+      replicas: 3
+      endpoint_mode: dnsrr
```
Comment (BenyH) [inline Setup/docker-compose_dev.yml]: what does it mean?
Do we need it?

File: Setup/docker-compose_dev.yml [hunk 4]
```diff
@@ -98,12 +97,37 @@ services:
...
+            failure_action: rollback
+        restart_policy:
+            condition: any
+        resources:
+          limits:
+            memory: 1GB
+            cpus: "1"
+      environment:
+          - "CONSUL_BIND_INTERFACE=eth2"
+          - "RUN_AGENT=yes"
```
Comment (BenyH) [inline Setup/docker-compose_dev.yml]: What Run_Agent means?
Please add a comment
Does RUN_AGENT is yes by default?
if this is the case, please comment it

**Comment by BenyH** [comment]
Hi Lev,
Please add describe what we are doing here
(adding an consul agent singletone)

**Comment by ozlevka-ericom** [comment]
Don't merge this pull request yet. I have a lot issues with this config

**Comment by ozlevka-ericom** [comment]
I close this pull request.
We have a lot of bad configurations here.
Big dangeros exists if changes will be merged to orid or dev config

**Pull request closed without merging**